example/simple/google_timestamp.swagger.json

```diff
@@ -17,6 +17,9 @@
   "host": "test.example.com",
   "paths": {},
   "definitions": {
+    "com.exampleEmpty": {
```

Comment:
Should we add a '.' separator between package / type here? `com.example.Empty` seems a more appropriate value?

Reply:
I am not sure what is correct here. Maybe `<package>/<message_name>`, so in this case it would be `com.example/Empty`. The 'old' swagger gen generated same output as this `<package><message_name>`.

Reply:
"import com.google.protobuf.Timestamp" existing in public code/docs leads me to believe that a dot is the established practice here. I'm gonna add an issue to implement this separately.